src/mongocxx/test/client.cpp

```diff
@@ -15,6 +15,10 @@
 #include "catch.hpp"
 #include "helpers.hpp"
 
+#define MONGOC_I_AM_A_DRIVER true
```

Comment:
This makes me a little sad. It will mean that packagers won't be able to build the tests against the system libmongoc unless there is also an unstable -priv package. How onerous would it be to do without?